drivers/hmis/app/graphql/types/hmis_schema/client_identifier.rb

```diff
@@ -0,0 +1,19 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::ClientIdentifier < Types::BaseObject
+    description 'Client Identifier'
```

Comment:
Can we rename this to `ExternalIdentifier`?
Other types of records will have external identifiers too (like Project) and we should be able to use the same type for them.